src/compiler/config/test/load-config.spec.ts

```diff
@@ -75,53 +102,44 @@ describe('load config', () => {
       expect(actualConfig.configPath).toBe(null);
     });
 
-    it('warns that a tsconfig file could not be found when "initTsConfig" set', async () => {
-      const loadedConfig = await loadConfig({ initTsConfig: true });
-
-      expect(loadedConfig.diagnostics).toHaveLength(1);
-      expect<d.Diagnostic>(loadedConfig.diagnostics[0]).toEqual({
-        absFilePath: '/tsconfig.json',
-        code: '18003',
-        columnNumber: undefined,
-        header: 'TypeScript',
-        language: 'typescript',
-        level: 'warn',
-        lineNumber: undefined,
-        lines: [],
-        messageText:
-          "No inputs were found in config file '/tsconfig.json'. Specified 'include' paths were '[\"src\"]' and 'exclude' paths were '[]'.",
-        relFilePath: undefined,
-        type: 'typescript',
-      });
+    it('creates a tsconfig file when "initTsConfig" set', async () => {
+      const tsconfigPath = path.resolve(path.dirname(noTsConfigPath), 'tsconfig.json');
+      expect(fs.existsSync(tsconfigPath)).toBe(false);
+      const loadedConfig = await loadConfig({ initTsConfig: true, configPath: noTsConfigPath });
+      expect(fs.existsSync(tsconfigPath)).toBe(true);
+      expect(loadedConfig.diagnostics).toHaveLength(0);
     });
 
     it('errors that a tsconfig file could not be created when "initTsConfig" isn\'t present', async () => {
-      const loadedConfig = await loadConfig({});
+      // const loadedConfig = await loadConfig({});
+      const loadedConfig = await loadConfig({ configPath: noTsConfigPath });
       expect(loadedConfig.diagnostics).toHaveLength(1);
       expect<d.Diagnostic>(loadedConfig.diagnostics[0]).toEqual({
         absFilePath: null,
         header: 'Missing tsconfig.json',
         level: 'error',
         lines: [],
-        messageText:
-          'Unable to load TypeScript config file. Please create a "tsconfig.json" file within the "/" directory.',
+        messageText: `Unable to load TypeScript config file. Please create a "tsconfig.json" file within the "./${path.dirname(
+          noTsConfigPath
+        )}" directory.`,
         relFilePath: null,
         type: 'build',
       });
     });
   });
 
   describe('no initialization argument', () => {
-    it('errors that a tsconfig file could not be created', async () => {
+    it('errors that a tsconfig file cannot be found', async () => {
       const loadConfigResults = await loadConfig();
       expect(loadConfigResults.diagnostics).toHaveLength(1);
       expect<d.Diagnostic>(loadConfigResults.diagnostics[0]).toEqual({
         absFilePath: null,
         header: 'Missing tsconfig.json',
         level: 'error',
         lines: [],
-        messageText:
-          'Unable to load TypeScript config file. Please create a "tsconfig.json" file within the "/" directory.',
+        messageText: expect.stringMatching(
+          `Unable to load TypeScript config file. Please create a "tsconfig.json" file within the`
+        ),
```

Comment:
Will this not be the root (`/`) directory anymore?

Reply:
the way the tests are mocked out it will be like `/Users/username/Code/stencil/` and then something else - so we can't precisely match on the full string anymore